pkg/util/secretutil/secret.go

```diff
@@ -157,7 +158,7 @@ func GetSecretData(secretObjData []*v1alpha1.SecretObjectData, secretType corev1
 		if len(data.Key) == 0 {
 			return datamap, fmt.Errorf("key in secretObjects.data is empty")
 		}
-		file, ok := files[data.ObjectName]
+		file, ok := files[strings.TrimSpace(data.ObjectName)]
```

Comment:
Do we also need to trim space for `data.Key`?

Reply:
Updated the PR!